### PR TITLE
Movie Project - Nathalie Nordström

### PR DESCRIPTION
**Pull request by nathalienordstrom**
Link to my deployed project:
https://movie-project-nathalie.netlify.app
My reflections on how this week's project turned out:

Things I'd like to have clarified or explained in more detail:

### CODE REVIEW

File: code/src/App.js [hunk 1]
```diff
@@ -1,9 +1,30 @@
 import React from 'react'
+import { BrowserRouter, Switch, Route, Redirect} from 'react-router-dom'
+import { PopularList } from 'components/PopularList'
```
Comment (destinyleia) [inline code/src/App.js:3]: I actually learnt from my last code review that the outer dependencies (those located in node_modules) should be separated from local dependencies (all local files that we created like: components, functions, data such as arrays) with one empty space above/under each other. See the example under! 😁

import React from 'react'
import { BrowserRouter, Switch, Route, Redirect} from 'react-router-dom'

import { PopularList } from 'components/PopularList'
import { Detail } from 'components/Detail'
import { NotFound } from 'components/NotFound'

File: code/src/NotFound.css [hunk 2]
```diff
@@ -0,0 +1,118 @@
+
```
Comment (destinyleia) [inline code/src/NotFound.css:1]: Nice extra addition to the project! 😄

File: code/src/components/NotFound.js [hunk 3]
```diff
@@ -0,0 +1,26 @@
+import React from 'react'
+import { Link } from 'react-router-dom'
+
+
+export const NotFound = () => {
```
Comment (destinyleia) [inline code/src/components/NotFound.js:5]: Nice extra addition to the project! 😄

File: code/src/components/PopularList.js [hunk 4]
```diff
@@ -0,0 +1,33 @@
...
+
+export const PopularList = () => {
+    const [movies, setMovies] = useState([])
+
+    useEffect(() => {
+        fetch(`https://api.themoviedb.org/3/movie/popular?api_key=3c9451d52520351f0a27222d2fb5cbb2&language=en-US&page=1`)
+        .then((res) => res.json())
+        .then((json) => {
+            setMovies(json.results)
+            console.log(json)
```
Comment (destinyleia) [inline code/src/components/PopularList.js:13]: Before you’re done with the project you should delete all of the console.logs 😁

File: code/src/index.js [hunk 5]
```diff
@@ -1,6 +1,8 @@
 import React from 'react'
 import ReactDOM from 'react-dom'
 import './index.css'
```
Comment (destinyleia) [inline code/src/index.js:3]: Should be a space between import ` import ReactDOM from 'react-dom'` and import './index.css' 🥳

**Review by destinyleia — APPROVED** [review]
Love the extra details you have put down in this project Nathalie! 😄 
It really paid of! 😍

I'm so bad at commenting my code in the code but I understand that it's good for the ones who reads the code to write som commands just to understand how the person think, so I would love to see that more if you have extra time over. 😁

**Blue Level (Minimum Requirements)**
- [✅] Your app should have at least two pages - one showing a list of movies and one showing 
- [✅] You should follow the design from our example (but it's ok to change things - just try to make it look nice)

You have fullfilled all the Blue Level requirements & I APPPPROVE! 😍
Great Job Nathalie! ☀️

File: code/src/App.js [hunk 6]
```diff
@@ -1,9 +1,30 @@
...
+      <Detail />
+      </Route>
+      <Route path="/404">
+      <NotFound />
+      </Route>
+      <Redirect 
+      to="/404"
+      />
+    </Switch>
+    </BrowserRouter>
```
[the remark this answers is not in the document]
Comment (EmOlsz) [reply code/src/App.js:28]: 👍

File: code/src/components/Detail.js [hunk 7]
```diff
@@ -0,0 +1,42 @@
+import React, { useEffect, useState } from 'react'
+import { useParams } from 'react-router-dom'
+import { Link } from 'react-router-dom'
+import Icon from "./arrow.png";
```
Comment (EmOlsz) [inline code/src/components/Detail.js:4]: Uppercase names are reserved for components only, not images

File: code/src/components/Detail.js [hunk 8]
```diff
@@ -0,0 +1,42 @@
...
+    }, [movieId])
+
+    return (
+
+        <section className="movie-backdrop" style={{ backgroundImage: `url(https://image.tmdb.org/t/p/w1280${movies.backdrop_path})` }}>
+
+            <div className="movie-content">
+                <div className="back-movies">
+                    <Link to="/" className="link">
+                        <img src={Icon} style={{ width: "40px" }} alt="icon" />
```
Comment (EmOlsz) [inline code/src/components/Detail.js:26]: Why you added this style inside `style` attribute and not as a CSS class? Needs to be fixed

**Comment by nathalienordstrom** [comment]
now it should be okey

**Comment by EmOlsz** [comment]
Yep, it's ok now, approved!
One thing I would like to mention nonetheless - during next projects try to add more meaningful commit names, ok? :)

**Pull request closed without merging**